dbt/models/model/model.vw_pin_shared_input.sql

```diff
@@ -63,19 +63,23 @@ acs5 AS (
     WHERE geography = 'tract'
 ),
 
-/* This CTAS uses location.census_2010 rather than joining onto a specific year
-from location.census because we need to join 2010 PUMA geometry to *all*
-parcels, not just those that existed in 2010 (or, in our case, 2012 since we
-don't have 2010 PUMA shapefiles). This is specific to the IHS data since it
+/* This CTAS uses location.census_2020 rather than joining onto a specific year
+from location.census because we need to join 2020 PUMA geometry to *all*
+parcels, not just those that existed in 2020 (or, in our case, 2022 since we
+don't have 2020 PUMA shapefiles). This is specific to the IHS data since it
 exists for many years but uses static geography. */
 housing_index AS (
     SELECT
         puma.pin10,
         ihs.year,
+        -- This is quarterly data and needs to be averaged annually
         AVG(CAST(ihs.ihs_index AS DOUBLE)) AS ihs_avg_year_index
-    FROM {{ source('other', 'ihs_index') }} AS ihs
-    LEFT JOIN {{ ref('location.census_2010') }} AS puma
-        ON ihs.geoid = puma.census_puma_geoid
+    FROM (SELECT DISTINCT
+        pin10,
+        census_puma_geoid
+    FROM {{ ref('location.census_2020') }}) AS puma
+    LEFT JOIN {{ source('other', 'ihs_index') }} AS ihs
+        ON puma.census_puma_geoid = ihs.geoid
```

Comment:
**[Praise]** It's also faster, which is a nice bonus! I timed the original query at 3m39s, while the new query executes in 1m3s.

At the risk of being pedantic, there's one small way in which these changes alter the output of the CTE: Since `puma` is now the left side of the join, there are roughly 14k pin10s that are in the output with null IHS index values that didn't used to be there. This shouldn't make a difference in the downstream queries, though, since we left join to this CTE by pin10 and year anyway.